federation/pkg/kubefed/init/init.go

```diff
@@ -62,6 +63,16 @@ const (
 	AdminCN                     = "admin"
 	HostClusterLocalDNSZoneName = "cluster.local."
 
+	// User name used by federation controller manager to make
+	// calls to federation API server.
+	ControllerManagerUser = "federation-controller-manager"
+
+	// Name of the ServiceAccount used by the federation controller manager.
```

Comment:
Please clarify in the comment is the SA used to talk to federation apiserver or to talk to registered clusters or just the host cluster or all of them :)